src/nyc_trees/apps/quiz/models.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+
+class Quiz(object):
```

Comment:
These models used to live in a module called `quizzes.py` but I moved it here to mimic how achievements are defined (Ref: https://github.com/azavea/nyc-trees/blob/develop/src/nyc_trees/apps/users/models.py#L15)